backend/docker-compose.yml

```
@@ -12,61 +11,43 @@ services:
-      db:
+      postgres:
         condition: service_healthy
     restart: on-failure
-    
+
 volumes:
-  pgdata:
-  pgadmin_data:
-  minio-data:
+  postgres-data:
+  pgadmin-data:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)